WMCDuplicateRemover/Code/Wrappers/MicrosoftEventLogWrapper.cs

```diff
@@ -36,8 +34,8 @@ public override bool FoundEventForRecording(String seriesName, String episodeNam
             //These are exceptions since I had to go and delete off these because WMC screwed up its DRM and I wasn't actually
             //finished watching these
             //TODO: Add the ability to add exceptions to this program without hard coding here
```

Comment:
Yeah, potentially something in an external file that it can look at or the app.config. I'd probably lean towards just a comma separated external file to make it more user friendly than modifying the app.config